test/hotspot/jtreg/runtime/8176717/TestInheritFD.java

```diff
@@ -187,10 +188,9 @@ static void fakeLeakyJVM(boolean fake) {
 
     static Collection<String> outputContainingFilenames() {
         long pid = ProcessHandle.current().pid();
-
         String[] command = lsofCommand().orElseThrow(() -> new RuntimeException("lsof like command not found"));
-        System.out.println("using command: " + command[0] + " " + command[1]);
-        return run(command[0], command[1], "" + pid).collect(toList());
+        System.out.println("using command: " + command[0] + " -a +d " + USER_DIR + " " + command[1] + " " + pid);
+        return run(command[0], "-a", "+d", USER_DIR, command[1], "" + pid).collect(toList());
```

Comment:
You might want to add a comment above L192:
// Only search the directory in which the VM is running (user.dir property):